README.md

```diff
@@ -176,7 +200,20 @@ The library is at `build_cmake/libCouchbaseLiteC.so`. (Or `.DLL` or `.dylib`)
 
 ### With CMake on Windows
 
-_(Much like building on Unix. Details TBD)_
+Dependencies:
+* [GCC 7+](http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe/download)
```

Comment:
I'll defer to @borrrden, but GCC shouldn't be necessary on Windows, since we support the MSVC compiler.

Reply:
And _only_ the MSVC compiler.  MingW (and it's even worse cousin Cygwin) adds an additional layer of headaches and dependencies that most of the time add no benefit.